java/vector/pom.xml

```diff
@@ -112,7 +109,7 @@
             <phase>test</phase>
             <configuration>
               <classpathDependencyExcludes>
-                <classpathDependencyExclude>org.apache.arrow:arrow-memory-unsafe</classpathDependencyExclude>
+                <classpathDependencyExclude>org.apache.arrow:arrow-memory-netty</classpathDependencyExclude>
```

Comment:
I am wondering if we need to keep this exclusion(for transitive dependency) as we have removed dependency on memory netty [here](https://github.com/sirensolutions/arrow/pull/50/files#diff-420770b87624123b845c55ec3b2e2e953e45b87be338efe313fa827303d9e754L56)?

Reply:
Since we are removing siren netty, this should be reverted to the original `unsafe`

Reply:
The tag `<classpathDependencyExclude>` would remove the dependency and we are trying to exclude netty here and keep unsafe?

Reply:
what we want to do is to remove our fork of netty but keep the default behavior of arrow with respect to netty

Reply:
This has now been reverted back in 9a6b412cffb43d540a98596bb0fe841d3434baa8